### PR TITLE
HDR texture exporting

### PR DESCRIPTION
**Pull request by kristiker**
Exporting vtex_c textures with HDR content to EXR file format.

### CODE REVIEW

File: GUI/Types/Renderer/GLTextureViewer.cs [hunk 1]
```diff
@@ -342,16 +343,16 @@ private void AddChannelsComboBox()
         private void SetInitialDecodeFlagsState(CheckedListBox listBox)
         {
             listBox.Items.Clear();
-            var values = Enum.GetValues<TextureCodec>();
+            var values = Enum.GetValues<TextureCodec>()[1..^1]; // Skip None, Auto
```
Comment (xPaw) [inline GUI/Types/Renderer/GLTextureViewer.cs:346]: feels like magic that will break eventually

File: GUI/Types/Renderer/GLTextureViewer.cs [hunk 2]
```diff
@@ -781,19 +783,21 @@ private void UploadTexture(bool forceSoftwareDecode)
...
             Debug.Assert(bitmap != null);
-            Debug.Assert(bitmap.ColorType == SKColorType.Bgra8888);
 
             texture = new RenderTexture(TextureTarget.Texture2D, bitmap.Width, bitmap.Height, 1, 1);
-            decodeFlags = TextureCodec.None;
+            decodeFlags &= softwareDecodeOnlyOptions;
 
-            GL.TextureStorage2D(texture.Handle, 1, SizedInternalFormat.Rgba8, texture.Width, texture.Height);
-            GL.TextureSubImage2D(texture.Handle, 0, 0, 0, texture.Width, texture.Height, PixelFormat.Bgra, PixelType.UnsignedByte, bitmap.GetPixels());
+            var isHdr = bitmap.ColorType == SKColorType.RgbaF32;
```
Comment (xPaw) [inline GUI/Types/Renderer/GLTextureViewer.cs:796]: not textureData.IsHighDynamicRange?

Comment (kristiker) [reply GUI/Types/Renderer/GLTextureViewer.cs:796]: GenerateBitmap can return LDR bitmap with TextureCodec.ForceLdr

Comment (kristiker) [reply GUI/Types/Renderer/GLTextureViewer.cs:798]: This reminds me that the HW decoder does not respect that flag.

**Pull request merged**